src/app/Components/ArtworkRail/ArtworkRailCard.tsx

```diff
@@ -63,6 +80,13 @@ export const ArtworkRailCard: React.FC<ArtworkRailCardProps> = ({
     ;(artwork as DissapearableArtwork)?._disappearable?.disappear()
   }
 
+  // 36 = 20 (padding) + 16 (icon size) + 5 (top padding)
+  const likeAndFollowCTAPadding =
+    enableNewSaveAndFollowOnArtworkCard && (enableNewSaveCTA || enableNewSaveAndFollowCTAs) ? 41 : 0
+  const artworkRailCardMetaPadding = 10
+  const artworkRailCardMetaDataHeight =
+    ARTWORK_RAIL_TEXT_CONTAINER_HEIGHT + artworkRailCardMetaPadding + likeAndFollowCTAPadding // TODO: check height
```

Comment:
do you still need to check the height here 👀 
```suggestion
    ARTWORK_RAIL_TEXT_CONTAINER_HEIGHT + artworkRailCardMetaPadding + likeAndFollowCTAPadding // TODO: check height
```

Reply:
Nope 👍